extensions/cornerstone/src/utils/WarningsViewportOverlay.js

```diff
@@ -0,0 +1,236 @@
+import { PureComponent } from 'react';
+import React from 'react';
+import PropTypes from 'prop-types';
+import cornerstone from 'cornerstone-core';
+import './WarningsViewportOverlay.css';
+import moment from 'moment';
+import classNames from 'classnames';
+import { Icon } from '@ohif/ui/src/elements/Icon';
+import { Tooltip } from '@ohif/ui/src/components/tooltip';
+import { OverlayTrigger } from '@ohif/ui/src/components/overlayTrigger';
+
+function isValidNumber(value) {
+  return typeof value === 'number' && !isNaN(value);
+}
+
+function formatNumberPrecision(number, precision) {
```

Comment:
All these formatting and checking functions fits well in utils though.

Reply:
done